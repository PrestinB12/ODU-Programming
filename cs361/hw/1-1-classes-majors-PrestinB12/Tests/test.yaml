test:
  timelimit: 10
  launch: ./listMajor
  stderr: false
dueDate: 09/07/2023
latePenalties: [10, 20, 100]
build:
  command: make
  weight: 1
  studentFiles:
    include: ["readingList.h", "readingList.cpp"]
  instructorFiles:
    exclude: ["readingList.h", "readingList.cpp"]
