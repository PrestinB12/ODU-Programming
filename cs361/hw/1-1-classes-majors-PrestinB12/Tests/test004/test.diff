22d21
< **Memory leaks detected: 200
