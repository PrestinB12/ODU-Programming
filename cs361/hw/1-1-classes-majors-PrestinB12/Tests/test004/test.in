math101  
bio101 
geo101
anth201 
psyc314
chem287
cs109
econ241
eng101
eng319
eng215
mgmt284
psci392
span103
fren103
ling101
art222
mus482
phil401
soc101
