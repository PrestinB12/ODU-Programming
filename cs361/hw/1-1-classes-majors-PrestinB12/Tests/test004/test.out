Semester 1
   anth201 ()
   art222 ()
   bio101 ()
   chem287 ()
   cs109 ()
   econ241 ()
   eng101 ()
   eng215 ()
   eng319 ()
   fren103 ()
   geo101 ()
   ling101 ()
   math101 ()
   mgmt284 ()
   mus482 ()
   phil401 ()
   psci392 ()
   psyc314 ()
   soc101 ()
   span103 ()
**Memory leaks detected: 200
